build/run-wasm-rego-tests.sh

```diff
@@ -44,7 +44,6 @@ function generate_testcases {
     purge_testgen_container
     docker run \
         --name $TESTGEN_CONTAINER_NAME \
-        -u $(id -u):$(id -g) \
```

Comment:
Wont this break linux users where the generated files will be owned by root? (osx does special user permission mapping with the shared files into the docker vm)

Reply:
ah, I didn't realize that Docker for Mac was doing some magic. OK, I'll figure something out.